README.md

```diff
@@ -14,109 +14,106 @@ Heat is a distributed tensor framework for high performance data analytics.
+[![Anaconda-Server Badge](https://anaconda.org/conda-forge/heat/badges/version.svg)](https://anaconda.org/conda-forge/heat)
 [![fair-software.eu](https://img.shields.io/badge/fair--software.eu-%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F-green)](https://fair-software.eu)
 [![OpenSSF Best Practices](https://bestpractices.coreinfrastructure.org/projects/7688/badge)](https://bestpractices.coreinfrastructure.org/projects/7688)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.2531472.svg)](https://doi.org/10.5281/zenodo.2531472)
 [![Benchmarks](https://img.shields.io/badge/Github--Pages-Benchmarks-2ea44f)](https://helmholtz-analytics.github.io/heat/dev/bench)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-# Goals
+- Table of Contents
+  - [What is Heat for?](#what-is-heat-for)
+  - [Features](#features)
+  - [Installation](#installation)
+    - [Requirements](#requirements)
+    - [pip](#pip)
+    - [conda](#conda)
+  - [Getting Started](#getting-started)
+  - [FAQ](#faq)
+  - [Support Channels](#support-channels)
+  - [Contribution guidelines](#contribution-guidelines)
+    - [Resources for MPI programming](#resources-for-mpi-programming)
+  - [License](#license)
+  - [Citing Heat](#citing-heat)
+  - [Acknowledgements](#acknowledgements)
 
-Heat is a flexible and seamless open-source software for high performance data
-analytics and machine learning. It provides highly optimized algorithms and data
-structures for tensor computations using CPUs, GPUs, and distributed cluster
-systems on top of MPI. The goal of Heat is to fill the gap between data
-analytics and machine learning libraries with a strong focus on single-node
-performance, and traditional high-performance computing (HPC). Heat's generic
-Python-first programming interface integrates seamlessly with the existing data
-science ecosystem and makes it as effortless as using numpy to write scalable
-scientific and data science applications.
 
-Heat allows you to tackle your actual Big Data challenges that go beyond the
-computational and memory needs of your laptop and desktop.
+# What is Heat for?
 
-# Features
+Heat builds on [PyTorch](https://pytorch.org/) and [mpi4py](https://mpi4py.readthedocs.io) to provide high-performance computing infrastructure for memory-intensive applications within the NumPy/SciPy ecosystem.
 
-* High-performance n-dimensional tensors
-* CPU, GPU, and distributed computation using MPI
-* Powerful data analytics and machine learning methods
-* Abstracted communication via split tensors
-* Python API
 
-# Support Channels
+With Heat you can:
+- port existing NumPy/SciPy code from single-CPU to multi-node clusters with minimal coding effort;
+- exploit the entire, cumulative RAM of your many nodes for memory-intensive operations and algorithms;
+- run your NumPy/SciPy code on GPUs (CUDA, ROCm, coming up: Apple MPS).
 
```

Comment:
@mrfh92 at this point, how about inserting a code snippet, and one or two of your hSVD plots?

Reply:
I have inserted a link to our blog post in the github pages.